client/src/components/SupermarketDisplay.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "@emotion/styled";
+
+function SupermarketDisplay({ supermarket }) {
+  return (
+    <Container>
+      <div>
+        <h3>{supermarket.name}</h3>
+      </div>
+      <div>
+        <p>
+          Products found: {supermarket.matchingProductsFound} of{" "}
+          {supermarket.listProductCount}
+        </p>
+        <p>{supermarket.totalPrice.toFixed(2)}€</p>
+      </div>
+    </Container>
+  );
+}
+
+export default SupermarketDisplay;
+SupermarketDisplay.propTypes = {
+  supermarket: PropTypes.any,
```

Comment:
```suggestion
  supermarket: PropTypes.object,
```
If i pass other than an object to SupemarketDisplay, the App breaks, because a string has no property "name"

Reply:
@mpagels see https://github.com/JonasImm/mega-app2020/pull/20#discussion_r488724430 for another solution

Reply:
thx